workspaces/acs/plugins/acs/README.md

```diff
@@ -0,0 +1,138 @@
+# Advanced Cluster Security Backstage Plugin
+
+![ACS plugin image 1](images/acs_plugin_screenshot_1.png)
+![ACS plugin image 2](images/acs_plugin_screenshot_2.png)
+
+## For Administrators
+
+### Backstage Installation
+
+Run the following command to install the ACS Backstage provider plugin:
+
+```console
+yarn --cwd packages/app add @backstage-community/plugin-acs
+```
+
+### Entity Annotation
+
+to have a comma separated string of deployment names from the ACS API you wish to test.
```

Comment:
I think maybe this should be deleted, might just be a copy/paste typo?

Reply:
Ugh, yeahhh that was..